helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -64,6 +64,7 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--timeout=180s"
```

Comment:
attacher timeout should be configurable

Reply:
Hold on to this, let the status of the other comment by @datamattsson get decided before we pursue this.

Reply:
@raunakkumar @datamattsson - What about the attacher timeout, this timeout is different for nimble and 3par. Do we make it configurable so you get to choose different timeouts based on whether it is nimble or 3par ?
If it is both nimble and 3par, we need to be on the higher side, which is 180s

Reply:
@raunakkumar - For OCP, the customer usually installs from the market place. Do you know of a way , that we can tweak this value , before installing from market place. Orange customer needs education on this front as they do not know how to tweak these params before installation.

Reply:
@pavansshanbhag don't mention customer names in public repos.

Reply:
This PR is open from last few days and we need to decide faster. My proposal is to make this configurable, so that both nimble and 3par can have the values that works for them. The default value for attachertimeout is 15 seconds, and i can maintain that as it is, if this parameter is not configured.
Let me know what you think.

Reply:
+1 to keep it configurable until we split the charts so that the defaults work as is and there is no regression.

Reply:
Turn this into a configurable parameter of the chart (`csiAttacherTimeout`) and leave the shipping default as the chart default.

Reply:
> @pavansshanbhag don't mention customer names in public repos.

Removed

Reply:
> Turn this into a configurable parameter of the chart (`csiAttacherTimeout`) and leave the shipping default as the chart default.

Agreed

Reply:
> +1 to keep it configurable until we split the charts so that the defaults work as is and there is no regression.

Agreed

Reply:
modified